editor/src/document/document_message_handler.rs

```diff
@@ -1100,7 +1101,8 @@ impl MessageHandler<DocumentMessage, (&InputPreprocessorMessageHandler, &FontCac
 				}
 			}
 			FolderChanged { affected_folder_path } => {
-				let _ = self.graphene_document.render_root(self.view_mode, font_cache);
+				// TODO: WHY IS IT RENDERING HERE? Can the line below be removed?
+				let _ = self.graphene_document.render_root(self.view_mode, font_cache, None);
```

Comment:
I think it can be removed - it doesn't do anything except from updating any dirty layers svg cache (which would happen at some point anyway)